### PR TITLE
feat(lb-properties): add new lb properties

### PR DESCRIPTION
**Pull request by var2dan**
ref #2993

### CODE REVIEW

**Comment by var2dan** [comment]
@jzaefferer the tests are now passing

**Comment by jzaefferer** [comment]
Need to also test with YAML input. Preferably with a unit test.

File: pkg/api/apps.go [hunk 1]
```diff
@@ -14,22 +14,23 @@ type AppsEndpoint struct {
...
-	Dependencies []string          `json:"dependencies,omitempty"`
-	EnvVars      map[string]string `json:"env,omitempty"`
-	Volumes      []*Volume         `json:"volumes,omitempty"`
-	HealthChecks []*HealthCheck    `json:"health_checks,omitempty"`
-	Logging      *Logging          `json:"logging,omitempty"`
+	ID                   *string           `json:"id,omitempty"`
+	Status               []string          `json:"status,omitempty"`
+	Domain               *Domain           `json:"domain,omitempty"`
+	SSL                  *bool             `json:"ssl,omitempty"`
+	ForceRollingDeploy   *bool             `json:"ssl,omitempty"`
```
Comment (jzaefferer) [inline pkg/api/apps.go]: Wrong property on the right, need to replace `ssl` with `forceRollingDeploy`

File: command/decoder_test.go [hunk 2]
```diff
@@ -714,6 +742,16 @@ services:
...
 `
 
+var testLbYAMLInput = `version: "v1"
+project: "wordpress"
+services:
+  frontend:
+    apache:
+      image: "wordpress:4.2"
+      domain:
+        redirectHttps: "true"
```
Comment (jzaefferer) [inline command/decoder_test.go]: Why are both of these boolean values defined as strings?

**Pull request merged**